packages/client/src/authentication/Authentication.ts

```diff
@@ -176,6 +176,22 @@ export class Authentication implements IAuthentication {
     return handler(profileId);
   }
 
+  async requireOnlyWalletAuthentication<
```

Comment:
this is more like `requireAnyAuthentication` where the other method is `requireProfileAuthentication` yes?

Reply:
good point, thanks